src/notation/view/widgets/editstyle.ui

```diff
@@ -226,7 +235,7 @@
      </widget>
      <widget class="QStackedWidget" name="pageStack">
       <property name="currentIndex">
-       <number>0</number>
+       <number>3</number>
```

Comment:
Might be a harmless change, as IIRC it does get set programatically to 0 enaway, but better lear it at 0.

Reply:
this is so weird ... why would it changed to 3

Reply:
because that was the tab you last worked on

Reply:
it is gone now